### ISSUE TITLE
Unable to rename a file via ftp using winscp

### ISSUE THREAD

**Issue by cracklejoy** [aldostools/webMAN-MOD#1035]
Attempting to rename any file via ftp results in the following error and a disconnect from the ftp server.

Error in data connection
Can't get attributes of file ...

webMAN MOD v1.47.45
WinSCP v6.3.1

**Comment by aldostools** [comment]
Try using FileZilla. It's the recommended FTP Client

**Comment by cracklejoy** [comment]
While filezilla may work this is still a bug.

**Comment by aldostools** [comment]
Right... you can report the bug to WinSCP or submit a PR with the correction to this module
https://github.com/aldostools/webMAN-MOD/blob/master/include/ftp.h
